src/components/Card.js

```diff
@@ -62,10 +87,39 @@ const Card = (props) => {
             <SocialLinks links={person.social_networks} />
             <City>{person.city}</City>
             <Buttons>
-                { !person.isContact && <Button variant="outlined" size="small" color="primary" className={classes.button} >Add to contacts</Button>}
-                { person.isContact && <Button variant="outlined" size="small" style={{color: 'red', borderColor: 'red'}} className={classes.button}>Delete from contacts</Button>}
-                { !person.isFavourite && <Button variant="outlined" size="small" color="primary" className={classes.button}>Add to favourites</Button>}
-                { person.isFavourite && <Button variant="outlined" size="small" style={{color: 'red', borderColor: 'red'}} className={classes.button}>Delete from favourites</Button>}
+                { !isContact && 
+                    <Button 
+                        variant="outlined" 
+                        size="small" 
+                        color="primary" 
+                        className={classes.button}
+                        onClick={addToContactsHandler}
+                    >Add to contacts</Button>
+                }
+                { isContact && 
+                    <Button 
+                        variant="outlined" 
+                        size="small" 
+                        style={{color: 'red', borderColor: 'red'}} 
+                        className={classes.button}
+                        onClick={removeFromContactsHandler}
+                    >Delete from contacts</Button>}
+                { !isFavourite && 
+                    <Button 
+                        variant="outlined" 
+                        size="small" 
+                        color="primary" 
+                        className={classes.button}
+                        onClick={addToFavouritesHandler}
+                    >Add to favourites</Button>}
+                { isFavourite && 
```

Comment:
I think we should check if it is also isContact === true.
Because we allow user to add favorite only if people already in our contact.